File: erettsegi_emelt/2013_may_eng/c#/Feladat.cs
public class Feladat{
    public readonly string kerdes;
    public readonly string temakor;
    public readonly int pont;
    public readonly int valasz;

    public Feladat(string quest, int answ, int points, string cat) {
        kerdes = quest;
        valasz = answ;
        pont = points;
        temakor = cat;
    }
}

File: erettsegi/emelt/2013_may_eng/java/Feladat.java
public class Feladat{

    public final String kerdes;
    public final String temakor;
    public final int pont;
    public final int valasz;

    public Feladat(String firstLine, String secondLine) {
        var secondLineSplit = secondLine.split(" ");

        kerdes = firstLine;
        valasz = Integer.parseInt(secondLineSplit[0]);
        pont = Integer.parseInt(secondLineSplit[1]);
        temakor = secondLineSplit[2];
    }
}